internal/deviations/deviations.go

```diff
@@ -625,3 +625,10 @@ func ISISLspMetadataLeafsUnsupported(dut *ondatra.DUTDevice) bool {
 func QOSQueueRequiresID(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetQosQueueRequiresId()
 }
+
+// GetEcnSameMinMaxThresholdUnsupported returns true for devices that don't support Minimum and Maximum threshold values are same
```

Comment:
The comment of Go function should start with the function name, i.e., "EcnSameMinMaxThresholdUnsupported returns..." in this case.

Reply:
corrected